back-end/developerverse/models.py

```diff
@@ -1,3 +1,29 @@
+from django.conf import settings
 from django.db import models
 
 # Create your models here.
+
+class Project(models.Model):
+
+    # We cannot directly use 
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    name = models.CharField(
+        max_length=140
+    )
+    repo_url = models.URLField(
+        max_length=480, blank=True
+    ) # allow for longer URLs
+    project_url = models.URLField(
+        max_length=480, 
+        blank=True
+    )

```

Comment:
There's inconsistency in formatting. Are we keeping every property on a new line? 
My recommendation is, unless the line is 80+ characters, keep each property on one line.